src/Features/Core/Portable/CodeRefactorings/MoveType/AbstractMoveTypeService.MoveTypeEditor.cs

```diff
@@ -138,31 +138,39 @@ private IEnumerable<SyntaxNode> GetMembersToRemove(SyntaxNode root)
 
                 // get potential namespace, types and members to remove.
                 var removableCandidates = root
-                    .DescendantNodes(n => DescendIntoChildren(n, spine.Contains(n)))
-                    .Where(n => FilterToTopLevelMembers(n, State.TypeNode));
+                    .DescendantNodes(n => spine.Contains(n))
```

Comment:
Yes.  Ther was no need to have a helper function here.  We just always want to descend as long as we're on the spine, and never descend otherwise.   While descnding on the spine, we may hit things we don't care about (like type arguments on a class type).  So we then filter just to top level items.